Cleanup after the Otel upgrade (#4359)
grafana/agent: docs/sources/flow/reference/components/otelcol.processor.batch.md

`otelcol.processor.batch` accepts telemetry data from other `otelcol`
components and places them into batches. Batching improves the compression of
data and reduces the number of outgoing network requests required to transmit
data. This processor supports both size and time based batching.

> It is highly recommended to configure the batch processor on every Agent which 

Reviewer: ```suggestion
We strongly recommend that you configure the batch processor on every Agent that
```

Suggestion based on wording used in other Grafana docs.
